samples/config/prod/m-cluster/ctsstore.yaml

```diff
@@ -20,7 +20,7 @@ opendjJavaArgs: >
 podAntiAffinity: "hard"
 
 gcs:
-  backup: false
+  backup: true
```

Comment:
Nice. Thanks.

Reply:
What about creating the NFS backup volumes? Does this get done automatically? Any changes needed to gke-up.sh for this?